mmdet3d/models/dense_heads/shape_aware_head.py

```diff
@@ -0,0 +1,567 @@
+            layers. Default: (64, 64, 64).
+        shared_conv_strides (tuple): Strides for shared convolutional \
+            layers. Default: (2, 1, 1).
+        use_direction_classifier (bool): Whether to use direction classifier.
+            Default: True.
+        conv_cfg (dict): Config of conv layer.
+            Default: dict(type='Conv2d')
+        norm_cfg (dict): Config of norm layer.
+            Default: dict(type='naiveSyncBN2d', eps=1e-3, momentum=0.01).
+        bias (bool|str): Type of bias. Default: False.
+    """
+
+    def __init__(self,
+                 num_cls,
+                 num_base_anchors,
+                 box_code_size,
+                 in_channels,
+                 shared_conv_channels=(64, 64, 64),
+                 shared_conv_strides=(2, 1, 1),
+                 use_direction_classifier=True,
+                 conv_cfg=dict(type='Conv2d'),
+                 norm_cfg=dict(type='naiveSyncBN2d', eps=1e-3, momentum=0.01),
+                 bias=False):
+        # Heavier head and smaller feature map than SmallHead
+        # Achieved by adding more shared conv layers
+        super(LargeHead,
+              self).__init__(num_cls, num_base_anchors, box_code_size,
+                             in_channels, shared_conv_channels,
+                             shared_conv_strides, use_direction_classifier,
+                             conv_cfg, norm_cfg, bias)
+
+
+@HEADS.register_module()
+class HugeHead(BaseShapeHead):
+    """Head for huge objects in Shape Signature Network.
+
+    Args:
+        num_cls (int): Number of classes.
+        num_base_anchors (int): Number of anchors per location.
+        box_code_size (int): The dimension of boxes to be encoded.
+        in_channels (int): Input channels for convolutional layers.
+        shared_conv_channels (tuple): Channels for shared convolutional \
+            layers. Default: (128, 128, 64, 64, 64).
+        shared_conv_strides (tuple): Strides for shared convolutional \
+            layers. Default: (2, 1, 2, 1, 1).
+        use_direction_classifier (bool): Whether to use direction classifier.
+            Default: True.
+        conv_cfg (dict): Config of conv layer.
+            Default: dict(type='Conv2d')
+        norm_cfg (dict): Config of norm layer.
+            Default: dict(type='naiveSyncBN2d', eps=1e-3, momentum=0.01).
+        bias (bool|str): Type of bias. Default: False.
+    """
+
+    def __init__(self,
+                 num_cls,
+                 num_base_anchors,
+                 box_code_size,
+                 in_channels,
+                 shared_conv_channels=(128, 128, 64, 64, 64),
+                 shared_conv_strides=(2, 1, 2, 1, 1),
+                 use_direction_classifier=True,
+                 conv_cfg=dict(type='Conv2d'),
+                 norm_cfg=dict(type='naiveSyncBN2d', eps=1e-3, momentum=0.01),
+                 bias=False):
+        # Heavier head and smaller feature map than SmallHead
```

Comment:
Move this into docstring. May also discuss the difference between it and the `LargeHead`.